### PR TITLE
Fix assertions for inherited/owned properties of cssom-view/cssom-vie…

### LINKED ISSUE #5672: Investigate failure of cssom-view/cssom-view/media-query-list-interface.xht
http://w3c-test.org/cssom-view/cssom-view/media-query-list-interface.xht fails for me in Gecko, Chromium and WebKit.

cc @zcorpan

### PR DESCRIPTION
**Pull request by fred-wang**
…w/media-query-list-interface.xht ; closes #5672

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5679)

### CODE REVIEW

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5679/)

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/225332716)

# Lint #

Passed

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/225332718)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 8da00a99e7dbb8108412ca5a70fcec0ad18aed8b
Using browser at version BuildID 20170423100233; SourceStamp 070fc3c2f8400c09f1037f5444c3d33b499e7128
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5679/cssom-view/cssom-view/media-query-list-interface.xht">/cssom-view/cssom-view/media-query-list-interface.xht</a></summary>


|               Subtest                | Results | Messages |
|--------------------------------------|---------|----------|
|                                      | OK      |          |
| `window_inherits_matchmedia`         | PASS    |          |
| `mediaquerylist_inherited_functions` | PASS    |          |
| `mediaquerylist_functions`           | PASS    |          |
| `mediaquerylist_properties`          | PASS    |          |
| `mediaquerylist_properties_readonly` | PASS    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/225332724)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 8da00a99e7dbb8108412ca5a70fcec0ad18aed8b

**Comment by fred-wang** [comment]
cc'ing @cdumez

File: cssom-view/cssom-view/media-query-list-interface.xht [hunk 1]
```diff
@@ -18,10 +18,10 @@
             var mediaQueryList;
             test(
                 function(){
-                    assert_inherits(window, "matchMedia");
+                    assert_own_property(window, "matchMedia");
                     mediaQueryList = window.matchMedia('foo');
                 }, 'window_inherits_matchmedia', {
-                  assert: ['window.matchMedia is inherited. If this fails, the rest of the test fails.']
+                  assert: ['window.matchMedia is own by window. If this fails, the rest of the test fails.']
```
Comment (cdumez) [inline cssom-view/cssom-view/media-query-list-interface.xht]: I would say "window.matchMedia is exposed."

LGTM otherwise.

**Comment by fred-wang** [comment]
@cdumez Thanks. Done.

**Comment by zcorpan** [comment]
@jugglingmike can you help with https://travis-ci.org/w3c/web-platform-tests/jobs/226284470 ? (I could restart it, but wanted to check if something needs debugging.)

**Review by zcorpan — APPROVED** [review]

**Comment by fred-wang** [comment]
I just pushed a similar fix to window-interface.xht in order to force travis to run again.

**Comment by jugglinmike** [comment]
Thanks for the heads up, @zcorpan. That's an entirely new kind of error for me. [We've inherited that job from the `tools` project import](https://github.com/w3c/web-platform-tests/pull/5554), so I'm wondering if it looks familiar to @gsnedders or @jgraham. Have either of you seen that before?

**Comment by gsnedders** [comment]
@jugglinmike that's bizarre; it implies the QuickCheck-like hypothesis was only able to generate seven instances in a second while rejecting none as invalid… that's silly slow. o_O

**Comment by jugglinmike** [comment]
@gsnedders is there anything we can do to debug further? Or, given that this is the first time you are seeing this, should we wait until it crops up again before investing time?

**Comment by gsnedders** [comment]
@jugglinmike IMO we should just wait until it crops up again; hopefully just some weird contention for CPU in the VM or something.

**Comment by jugglinmike** [comment]
Works for me!

**Pull request merged**